lib/mayaUsd/base/mayaUsd.h.src

```diff
@@ -44,3 +44,46 @@ namespace MAYAUSD_NS {
 #else
 #define MAYAUSD_NS_DEF MAYAUSD_VERSIONED_NS
 #endif
+
```

Comment:
I created a set of macros to delete the copy/move operators and then to test that the class is setup correctly. The same macros can be found in UsdUfe.